src/browser/screens/backpack/BackpackScreen.js

```diff
@@ -36,9 +36,9 @@ export default class Third2Screen extends React.PureComponent {
             <strong>Pack Your bag</strong>
           </h1>
         </div>
-        <BackpackList />
+        {/*<BackpackList />*/}
```

Comment:
If you don't plan to bring it back in the next 2 weeks (this PR is 3 weeks old, so I guess no) - Kill it, don't leave it in code